_schemaV2.graphql

```diff
@@ -4766,9 +4766,15 @@ input BulkArtworkFilterInput {
 }
 
 input BulkUpdateArtworksMetadataInput {
+  # Number of additional edition sets to be created for each artwork
+  additionalEditionSetsCount: Int
```

Comment:
ahh sorry, I should have raised this earlier, but we changed this param name based on a discussion we had yesterday (https://github.com/artsy/gravity/pull/19329).

Because the edition sets won't always necessarily be added (it will take the existing sets into account), we changed this from `additionalEditionSetsCount` to `editionSetsCount`

Reply:
🙇‍♀️

Reply:
adjusted!